### PR TITLE
Clean up prettier/eslint scripts

### PR DESCRIPTION
**Pull request by estrattonbailey**
This PR hoists prettier/eslint scripts to root `package.json` and removes scripts and config from individual `packages` in order to simplify things a bit. It also makes sure that all packages are linted (api wasn't).

Main motivation here originally was the Changesets PR #1513 where I'd like to run prettier over changelogs as they're generated. This makes that a little cleaner/easier.

prettier go brrrr

### CODE REVIEW

File: packages/api/docs/labels.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
 <!-- this doc is generated by ./scripts/docs/labels.mjs -->
 
-  # Labels
-  
-  This document is a reference for the labels used in the SDK.
+# Labels
```
Comment (estrattonbailey) [inline packages/api/docs/labels.md:3]: Kinda double work, but seems OK to me.
1. Codegen runs
2. On/before commit, run prettier to tidy it up

**Comment by ericvolp12** [comment]
If you're gonna do this, make sure to put it in a precommit hook or something so it stays clean.

**Comment by estrattonbailey** [comment]
> If you're gonna do this, make sure to put it in a precommit hook or something so it stays clean.

Oh yes 😏

File: packages/api/docs/labels.md (hunk 1 above)
Comment (dholms) [reply packages/api/docs/labels.md:3]: it'd be nice if codegen genned code that passed prettier 😛

**Comment by dholms** [comment]
I'm down for this 👍 

Altho I'm tempted to just exclude genned code for now

I'm not particularly fond of linting running on commit :/

But I'm down to make the CI linting check (`verify`) lint the entire repo, including schemas/codegen

**Comment by bnewbold** [comment]
Seems nice!

I would run prettier as part of the codegen process itself, specifically over output code.

Would be great to have the Makefile invoke this for `make fmt` / `make lint`.

Like dholms I prefer to check in CI and not do linting in a pre-commit hook, but i'm sort of guest in this particular repo so :shrug:. It does look like we don't actually enforce a lint in CI on this repo though!

**Comment by dholms** [comment]
oh yeah doing it in the codegen step makes a lot of sense 👍

File: packages/api/docs/labels.md (hunk 1 above)
Comment (estrattonbailey) [reply packages/api/docs/labels.md:3]: FWIW codegen does do some prettier, but apparently not well enough.

My feeling is that if it's code we check in and/or ship to npm, it should be formatted. We can improve the codegen CLI to handle this, but for now it doesn't hurt to do it here.

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+packages/api/src/client
+packages/bsky/src/lexicon
+packages/pds/src/lexicon
```
Comment (estrattonbailey) [inline .eslintignore:3]: Copied from the individual packages (except API which wasn't linted)

File: .eslintrc [hunk 3]
```diff
@@ -26,7 +26,11 @@
   "rules": {
     "no-var": "error",
     "prefer-const": "warn",
-    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
+    "no-misleading-character-class": "warn",
```
Comment (estrattonbailey) [inline .eslintrc:29]: There was a regex in the rich text code that this didn't like, but didn't seem worth fixing atm.

File: package.json [hunk 4]
```diff
@@ -9,8 +9,10 @@
     "node": ">=18"
   },
   "scripts": {
-    "verify": "pnpm -r --stream lint && pnpm -r --stream prettier",
-    "format": "pnpm -r --stream prettier:fix",
+    "lint:fix": "pnpm lint --fix",
+    "lint": "eslint . --ext .ts,.tsx",
+    "verify": "prettier --check . && pnpm lint",
+    "format": "prettier --write .",
```
Comment (estrattonbailey) [inline package.json:15]: All run from the root now instead of the individual packages

File: packages/aws/package.json [hunk 5]
```diff
@@ -13,12 +13,6 @@
     "directory": "packages/aws"
   },
   "scripts": {
-    "prettier": "prettier --check src/",
-    "prettier:fix": "prettier --write src/",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "pnpm lint --fix",
-    "verify": "run-p prettier lint",
-    "verify:fix": "pnpm prettier:fix && pnpm lint:fix",
```
Comment (estrattonbailey) [inline packages/aws/package.json:21]: Was able to remove a bunch of scripts in the packages since everything is run from the root now

**Pull request merged**